test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -13152,6 +13196,8 @@ class MoM(PBSService):
                        'PBS_MANAGER_SERVICE_PORT': '-R',
                        'PBS_HOME': '-d'}
 
+    revert_default_attrib = {'resources_available.ncpus': 1}
+
```

Comment:
Instead of checking this to 1, we can get ncpus values when we create nodes for the first time in setUp class. Not all systems will have ncpus value as 1 by default.